multisig/proposals/arbitrumone/vip-013/index.ts

```diff
@@ -0,0 +1,232 @@
+import { parseUnits } from "ethers/lib/utils";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { makeProposal } from "src/utils";
+
+const { arbitrumone } = NETWORK_ADDRESSES;
+
+export const COMPTROLLER_LIQUID_STAKED_ETH = "0x52bAB1aF7Ff770551BD05b9FC2329a0Bf5E23F16";
+
+export const wstETH = "0x5979D7b546E38E414F7E9822514be443A4800529";
+export const weETH = "0x35751007a407ca6FEFfE80b3cB397736D2cf4dbe";
+export const WETH = "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1";
+
+export const VwstETH = "0x9df6B5132135f14719696bBAe3C54BAb272fDb16";
+export const VweETH = "0x246a35E79a3a0618535A469aDaF5091cAA9f7E88";
+export const VWETH = "0x39D6d13Ea59548637104E40e729E4aABE27FE106";
+
+export const PSR = "0xF9263eaF7eB50815194f26aCcAB6765820B13D41";
+export const vTokenReceiver = "0x5A9d695c518e95CD6Ea101f2f25fC2AE18486A61";
+export const NEW_COMPTROLLER_IMPLEMENTATION = "0x49Aa45B8256DBdFE90232f88f9e676a8Ec55D286";
+
+export const COMPTROLLER_BEACON = "0x8b6c2E8672504523Ca3a29a5527EcF47fC7d43FC";
+const wstETH_ONE_JUMP_ORACLE = "0x748DeFdbaE5215cdF0C436c538804823b55D76bc";
+const weETH_ONE_JUMP_ORACLE = "0x09eA4825a5e2FB2CB9a44F317c22e7D65053ea9d";
+const CHAINLINK_wstETH_FEED = "0xB1552C5e96B312d0Bf8b554186F846C40614a540";
+const CHAINLINK_weETH_FEED = "0x20bAe7e1De9c596f5F7615aeaa1342Ba99294e12";
+
+const STALE_PERIOD_26H = 60 * 60 * 26; // 26 hours (pricefeeds with heartbeat of 24 hr)
+
+// IL configuration
+const vip013 = () => {
+  return makeProposal([
+    {
+      target: COMPTROLLER_BEACON,
+      signature: "upgradeTo(address)",
+      params: [NEW_COMPTROLLER_IMPLEMENTATION],
+    },
```

Comment:
We should remove this, because the needed change will be included with the upgrade performed in https://github.com/VenusProtocol/vips/pull/184

Reply:
I have pasted the VIP created by Narayan for arbitrumone to avoid the conflicts, and removed upgrade command from VIP-013. Simulations are pretending VIP-012(Upgrade one) then VIP-013.
44fd3cb978804fb1c7031e5684ccdb14a8b95b18